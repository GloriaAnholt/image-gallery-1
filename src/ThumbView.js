import React from 'react';

export default function ThumbView(props) {
// "thumbnail" - the image as a "thumbnail" (within a 100x100 pixel scale) and the title . 
  return (
    <h2>Thumb View</h2>
  )
}