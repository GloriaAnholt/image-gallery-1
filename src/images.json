[
  {
    "title": "Rainbow frog",
    "description": "I'm probably poisonious.",
    "url": "https://s-media-cache-ak0.pinimg.com/736x/bf/4e/40/bf4e4067252227bd3f758bba7dcee2ff.jpg"
  },
  {
    "title": "Green leaf frogs",
    "description": "We're so cute!",
    "url": "http://www.savethefrogs.com/amphibians/images/south-america/Hypsiboas-cinerascens-8a.jpg"
  },
  {
    "title": "Tree frog",
    "description": "Whaddya looking at buddy?",
    "url": "http://www.vorgee.com/sites/default/files/imagecache/left-col/Fun_Facts_Main_Frog.jpg"
  },
  {
    "title": "Toad",
    "description": "Bud. Bud. Bud.",
    "url": "http://www.defenders.org/sites/default/files/frog-kevin-clark-dpc.jpg"
  }
]