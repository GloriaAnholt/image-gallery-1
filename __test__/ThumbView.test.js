import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ThumbView from '../src/ThumbView';
import images from '../src/images.json'


describe('The Image Gallery React App', () => {

    it('compares a snapshot of the Thumb view', () => {
        const rendered = <ThumbView  images={ images }/>
        expect(toJson(rendered)).toMatchSnapshot();
    });

})