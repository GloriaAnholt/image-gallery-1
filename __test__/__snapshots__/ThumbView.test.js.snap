exports[`The Image Gallery React App compares a snapshot of the Thumb view 1`] = `undefined`;
